tokens/usecase/tokens_usecase.go

```diff
@@ -357,6 +363,64 @@ func (t *tokensUseCase) getChainScalingFactorMut(precision int) (osmomath.Dec, b
 	return result, ok
 }
 
+var (
+	jobIsRunning   bool
+	jobIsrunningMu sync.Mutex
+)
+
+// StartAsRoutineIfNotRunning starts the job in a new goroutine if it is not already running.
+func StartAsRoutineIfNotRunning(job func()) bool {
```

Comment:
Those functions are public, because tests resides in other package making original package to leak implementation details. Should I just create different file that would be within same package for example `tokens_usecase_i_test.go` where I could place tests?

Reply:
Removed this code, no longer relevant.